releasenotes-dca/notes/auto-instru-76b1d4f1b2ec1c84.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Experimental: The Datadog Admission Controller can inject the Node and Java APM libraries for auto-instrumentation.
```

Comment:
can we change the wording to "The Datadog Admission Controller can inject the Node and Java APM libraries into Kubernetes containers for auto-instrumentation."